Source/WebCore/crypto/keys/CryptoKeyOKP.h

```diff
@@ -37,7 +37,7 @@ class CryptoKeyOKP final : public CryptoKey {
 public:
     using KeyMaterial = Vector<uint8_t>;
 
-    enum class NamedCurve {
+    enum class NamedCurve : bool {
```

Comment:
This might add more types like `x448` and `ed448`. `uint8_t` should work.

Reply:
For IPC serialization, we prefer using `bool` for enum class that only contains two values as we could validate the values can only be 0 or 1 on receiver side. If we adds the values later, we could update it to uint8_t.